include/domain-types.h

```diff
@@ -215,14 +215,16 @@ struct ObjectStream
 
 struct Count
 {
-    Count(const timestamp_t& timestamp, int32_t value)
+    Count(const timestamp_t& timestamp, std::string& label, int32_t value)
         : timestamp(timestamp)
+        , label(label)
         , value(value)
     {
     }
 
     timestamp_t timestamp;
-    int32_t value;
+    std::string label;
+    int32_t     value;
```

Comment:
No need to change indent for value. Just leave one space as it was initially. There should be no diff for this line as a result